pycbc/io/live.py

```diff
@@ -508,6 +497,10 @@ def upload(self, fname, gracedb_server=None, testing=True,
                          if v != 0.0}
             labels, sizes = zip(*prob_plot.items())
             colors = [source_color(label) for label in labels]
+            import matplotlib
+            matplotlib.use('Agg')
```

Comment:
Please rebase on master and remove these two lines.

Reply:
Done